Gemfile.lock

```diff
@@ -76,7 +76,7 @@ DEPENDENCIES
   kramdown-parser-gfm
 
 RUBY VERSION
-   ruby 2.7.0p0
```

Comment:
Please don't change the ruby version

Reply:
changed that. please review it again, bhaiya.